WeakAuras/Types.lua

```diff
@@ -4298,7 +4323,7 @@ Private.glow_types = {
   buttonOverlay = L["Action Button Glow"],
 }
 
-if WeakAuras.IsMistsOrRetail() then
+if WeakAuras.IsWrathOrMistsOrRetail() then
   Private.glow_types.Proc = L["Proc Glow"]
```

Comment:
If I understood your direct messages correctly, this requires adding the retail file for the glow to our package.
We cannot do that, blizzard owns the copyright to that file.